source/vibe/core/path.d

```diff
@@ -189,6 +190,129 @@ string toNativeString(P)(P path)
 }
 
 
+/** Returns the file extension of a file, including the dot.
+
+	Note that `filename` must be a pure file name and not a full path.
```

Comment:
I don't understand why this function is not just part of `Path` ?

Reply:
Because that would also mean in `Segment2` and possibly `Segment`, including duplicate documentation, and skipping the range based overload (which ideally should be adjusted in Phobos, but anyway).